<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车轮旋转视觉效果</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="controlsWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QLabel" name="labelDelta">
          <property name="text">
           <string>转速增量 (RPM)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboDeltaRpm"/>
        </item>
        <item>
         <widget class="QLabel" name="labelRpm">
          <property name="text">
           <string>转速 (RPM)：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinRpm">
          <property name="minimum">
           <number>-4500</number>
          </property>
          <property name="maximum">
           <number>4500</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSpokes">
          <property name="text">
           <string>辐条数 (3-12)：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinSpokes">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFrequency">
          <property name="text">
           <string>采样频率 (Hz)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFrequency"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkColorMark">
          <property name="text">
           <string>颜色标记</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonToggle">
          <property name="text">
           <string>转动车轮</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="WheelWidget" name="wheelWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperate">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionToggle"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOperate"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>工具</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <addaction name="actionToggle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionToggle">
   <property name="text">
    <string>转动车轮</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WheelWidget</class>
   <extends>QWidget</extends>
   <header>wheelwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>